filters/auth/oidc_introspection.go

```diff
@@ -64,12 +64,12 @@ func (spec *oidcIntrospectionSpec) CreateFilter(args []interface{}) (filters.Fil
 	switch filter.typ {
 	case checkOIDCQueryClaims:
 		for _, arg := range sargs {
-			slice := strings.SplitN(arg, ":", 2)
-			if len(slice) != 2 {
+			path, queries, found := strings.Cut(arg, ":")
+			if !found || path == "" {
```

Comment:
The check for non-empty path might make sense but is out of scope for this refactoring. I suggest we keep existing logic here and elsewhere.